NEWS.md

```diff
@@ -1,7 +1,20 @@
 ## 0.5.2 to 0.5.3 (unreleased)
 
-(TODO)
-
+ * Events: Subqueries can be extended by the configuration, for example:
+   ```
+   "subqueries": {
+     "all_ips": {
+       "sql": "(\"source.ip\" = %s OR \"source.local_ip\" = %s OR \"destination.ip\" = %s OR \"destination.local_ip\" = %s)",
+       "description": "Queries (source|destination).(local_)ip",
+       "label": "Query all IPs",
+       "ext_type": "integer"
+     }
+   }
+   ```
+ * Events: Subqueries can need the parameter '%s' more or less than one time in the SQL query, for example:
```

Comment:
Suggestion for improvement: I'm trying to write the NEWS.md entries from a user perspective, maybe this sentence can be improved. I had to read it a few times to understand it. What about:
```
 * Events: Enhances subqueries, so they can use an SQL query that uses a single parameter several times.
```

To have queries without a parameter, I think we'd would also have to change the frontend, wouldn't we?